Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -41,6 +41,18 @@ void NSStringForceinclude() {
 
 @implementation NSString (UIKitAdditions)
 
+static CTTextAlignment __UITextAlignmentToCTTextAlignment(UITextAlignment alignment) {
```

Comment:
not like this - look at what I did in UIKitTypes.h for NS, UI, CT LineBreakMode. Then you can change it back to a static_cast below.